### ISSUE TITLE
Possible 'move item' contextual menu instead of drag and drop.

### ISSUE THREAD

**Issue by thiagodemellobueno** [kalamuna/kalaponents#59]
Not wanting to open recursively deep cans of worms, but there was talk of replacing some of the complexity of drag and drop mode with a "move to region" contextual menu ala invision:

<img width="719" alt="screen shot 2017-09-29 at 1 59 37 pm" src="https://user-images.githubusercontent.com/79221/31029105-794be864-a51e-11e7-90d6-00644d10b06e.png">

**Comment by thiagodemellobueno** [comment]
@derekderaps

**Comment by hawkeyetwolf** [comment]
Yes, I like this idea very much! It's little lower on the priority list since we have a "good enough" solution right now. But who wants "good enough"?? I want GREAT!
